### ISSUE TITLE
The required video memory for training.

### ISSUE THREAD

**Issue by fayewong666999** [Tele-AI/OmniVDiff#5]
OmniVDiff is an excellent work. May I ask what is the required amount of video memory for training?

**Comment by xdobetter** [comment]
Thanks for your interest!
In our setup, training OmniVDiff with `batch_size=1` requires approximately **33GB of GPU memory**.
If you use a smaller resolution or apply gradient checkpointing, you might be able to fit it in slightly less.
